### ISSUE TITLE
Inaccurate Pytorch comparison value?

### ISSUE THREAD

**Issue by MattWoolleySpeedShield** [drilistbox/TRT_FlashOcc#4]
after following all steps, my outputted PyTorch comparision accuracy value is only ~0.6 and seems no where near close to the 0.998 commented around, I have remade the models multiple times and even grabbed fresh onnx from a self made FlashOCC that worked using FlashOCC's method of running .engine files. Are there any working inference tools I can use to compare the outputted logits/labels to the input image (or the input file from the pytorch debug download 2) if you need any information for debugging just let me know! thanks
